discord/ext/converters/regex.py

```diff
@@ -0,0 +1,22 @@
+import re
+from typing import Optional
+
+from discord.ext import commands
+
+
+class MatchConverter(commands.Converter):
+    def __init__(self, pattern, flags) -> None:
+        self.pattern = pattern
+        self.flags = flags
+
+    async def convert(self, ctx, argument: str) -> re.Match:
+        match: Optional[re.Match] = re.fullmatch(
+            self.pattern, argument, flags=self.flags
```

Comment:
`flags` is not a keyword argument.

```suggestion
            self.pattern, argument, self.flags
```

Reply:
[It is](https://docs.python.org/3/library/re.html#re.fullmatch)?